src/main/resources/lang/UIText_pl.properties

```diff
@@ -0,0 +1,183 @@
+font.opensans.bold      = OpenSansBold+Cyberbit.ttf
+font.opensans.regular   = OpenSans+Cyberbit.ttf
+font.raleway.light      = Raleway+FireflySung.ttf
+
+updater.launcherupdate  = Aktualizacja launchera
+updater.mover           = Kopiowanie aktualizacji...
+updater.finallaunch     = Włączanie Technic...
+updater.launchmover     = Włączanie przenośnika aktualizacji...
+updater.downloads       = Pobieranie zasobów launchera: %s
+updater.folders         = Inicjalizacja katalogów launchera
+updater.query           = Sprawdzanie strumienia aktualizacji...
+
+launcher.installer.title    = Zainstaluj Technic
+launcher.installer.standard = Zwykła instalacja
+launcher.installer.portable = Wersja przenośna
+launcher.installer.standardspiel    = Zwykła instalacja jest najlepszą opcją dla większości użytkowników.  Zainstaluj na tym komputerze i odpal launcher z obojętnie jakiego katalogu.
+launcher.installer.portablespiel    = Wersja przenośna jest najlepsza dla użytkowników w ruchu.  Zainstaluj na pamięci przenośnej i odpal launcher z katalogu instalacyjnego.
+launcher.installer.install  = Instaluj
+launcher.installer.folder   = Folder instalacji
+launcher.installer.select   = Wybierz
+launcher.installer.default  = Wybierz domyślną lokalizację (Zalecane)
+installer.portable.replacefailed    = Zamknij wszystkie procesy launchera i spróbuj ponownie.
+installer.portable.replacefailtitle = Instalacja niepowiodła się
```

Comment:
Should be installer.portable.replacefailtitle = Instalacja nie powiodła się

that is with a space "nie powiodła"